astropy_helpers

```diff
@@ -1 +1 @@
-Subproject commit 90de2d8340d1eafaface009625056ca62f5a6c76
+Subproject commit 161773fa72d916c498e0a2a513ecc24460244ac8
```

Comment:
This `astropy_helpers` Git submodule change shouldn't be in this PR.
The easiest solution is to make a new commit that re-sets it to  90de2d8 which is the version pointing to in Gammapy master.

This should work:
```
cd astropy_helpers
git fetch origin
git checkout 90de2d8
cd ..
git add astropy_helpers
git commit -m "Re-set astropy-helpers to 90de2d8"
```